helm/prometheus-rules/templates/alerting-rules/etcdbackup.rules.yml

```diff
@@ -68,3 +68,15 @@ spec:
         severity: page
         team: {{ include "providerTeam" . }}
         topic: etcd-backup
+    - alert: ETCDBackupMetricsMissing
+      annotations:
+        description: '{{`ETCD backup metrics are missing`}}'
+        opsrecipe: etcd-backup-failed/
```

Comment:
It would be best to add a new section to the recipe, explaining that the metrics are exposed by the controller, so the oncaller needs to check the deployment status.

Reply:
https://github.com/giantswarm/giantswarm/pull/28825/files